snare/cloner.py

```diff
@@ -299,6 +294,13 @@ async def run(self):
     async def close(self):
         if not self.runner:
             raise Exception("Error initializing cloner!")
+        error_file_name, error_file_hash = self.runner._make_filename(self.runner.error_page)
```

Comment:
Do we really need an empty page? Why not letting aiohttp handle this?
https://docs.aiohttp.org/en/latest/web_exceptions.html

Reply:
Yeah I agree instead of generating a new empty page we should just let aiohttp handle that